tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1594,3 +1599,73 @@ Feature: get users
       }
     }
     """
+
+  @env-config
+  Scenario Outline: user searches user and see users email if ShowUserEmailInResults config is enabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "true"
```

Comment:
we should have coverage for similar scenario. We can just add example table there for different users
https://github.com/owncloud/ocis/blob/1d388a00a922e11ed5e8289e34295d9b09f3f5d5/tests/acceptance/features/apiGraphUserGroup/searchUserIncludingEmail.feature#L258

Reply:
done